examples/plot_alignment_methods_benchmark.py

```diff
@@ -0,0 +1,147 @@
+                  cut_coords=(8, -80, 9), colorbar=True, cmap='Paired')
+
+###############################################################################
+# Define a masker
+# ----------------
+# We define a nilearn masker that will be used to handle relevant data. \
+# For more information, visit : \
+# 'http://nilearn.github.io/manipulating_images/masker_objects.html'
+#
+
+from nilearn.input_data import NiftiMasker
+roi_masker = NiftiMasker(mask_img=resampled_mask_visual).fit()
+
+###############################################################################
+# Prepare the data
+# ---------------------------------------------------------------------------
+# For each subject, for each task and conditions, our dataset contains two \
+# independent acquisitions, similar except for one acquisition parameter, the \
+# encoding phase used that was either Antero-Posterior (AP) or \
+# Postero-Anterior (PA). Although this induces small differences \
+# in the final data, we will take  advantage of these pseudo-duplicates to \
+# create a training and a testing set that contains roughly the same signals \
+# but acquired totally independently.
+#
+
+# The training fold, used to learn alignment from source subject toward target:
+# * source train: AP contrasts for subject one
+# * target train: AP contrasts for subject two
+
+source_train = df[df.subject == 'sub-01'][df.acquisition == 'ap'].path.values
+target_train = df[df.subject == 'sub-02'][df.acquisition == 'ap'].path.values
+
+# The testing fold:
+# * source test: PA contrasts for subject one, used to predict \
+#   the corresponding contrasts of subject two
+# * target test: PA contrasts for subject two, used as a ground truth \
+#   to score our predictions
+
+source_test = df[df.subject == 'sub-01'][df.acquisition == 'pa'].path.values
+target_test = df[df.subject == 'sub-02'][df.acquisition == 'pa'].path.values
+
+###############################################################################
+# Choose the number of regions for local alignment
+# ---------------------------------------------------------------------------
+# First, as we will proceed to local alignment we choose a suitable number of \
+# regions so that each of them is approximately 200 voxels wide. Then our \
+# estimator will first make a functional clustering of voxels based on train \
+# data to divide them into meaningful regions.
+#
+
+import numpy as np
+n_voxels = roi_masker.mask_img_.get_data().sum()
+print("The chosen region of interest contains {} voxels".format(n_voxels))
+n_pieces = int(np.round(n_voxels / 200))
+print("We will cluster them in {} regions".format(n_pieces))
+
+###############################################################################
+# Define the estimators, fit them and do a prediction
+# ---------------------------------------------------------------------------
+# On each region, we search for a transformation R that is either :
```

Comment:
the term 'region' is for the whole; one should call the sub-regions pieces or parcels to avoid confusion